README.md

```diff
@@ -563,11 +563,11 @@ S3 configuration:
 
 AWS Authentication:
 
-The simplest way is to use an IAM Instance profile.
-Other options are:
+Buildkit rely on the [AWS Go SDK](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/config#EnvConfig). This means that all standard authentication methods through environments variables or config files are supported. This is espcially true for AWS EC2 IAM Profile and AWS Web Identity Token (IAM roles in Kubernetes). 
```

Comment:
"BuildKit relies"
"through environment variables"
"especially"

Reply:
Thx a lot. Sorry for my poor english.